bootstrap_anaconda

```diff
@@ -40,10 +40,13 @@ mirrorcurl() {
 
 rm -f .treeinfo
 mirrorcurl .treeinfo
-mirrorcurl images/pxeboot/vmlinuz
-mirrorcurl images/pxeboot/initrd.img
-mirrorcurl images/pxeboot/upgrade.img
-mirrorcurl LiveOS/squashfs.img
+
+img_files=(images/pxeboot/vmlinuz images/pxeboot/initrd.img images/pxeboot/upgrade.img LiveOS/squashfs.img)
```

Comment:
The functionality is nice.

what about creating a small bash func which takes the filename and the path as args before trying to download, i.e.:
```
if_in_tree_mirrorcurl() { $MAGIC_CHECK $1 && mirrorcurl $2 ; }
```